### PR TITLE
Reactadmin upgrade

### PR DESCRIPTION
**Pull request by ThomasDCW**

### CODE REVIEW

**Review by Dash293 — DISMISSED** [review]

File: backend/src/router.js [hunk 1]
```diff
@@ -45,6 +45,9 @@ router.put("/topics/:id", TopicController.edit);
 router.post("/topics", TopicController.add);
 router.delete("/topics/:id", TopicController.delete);
 router.get("/schoolclass", ClassController.browse);
+router.get("/schoolclass/:id", ClassController.read);
+router.get("/schoolclass/:id", ClassController.edit);
```
Comment (LuigiMenez) [inline backend/src/router.js:49]: Nop

File: backend/src/router.js [hunk 2]
```diff
@@ -45,6 +45,9 @@ router.put("/topics/:id", TopicController.edit);
...
+router.get("/schoolclass/:id", ClassController.delete);
```
Comment (LuigiMenez) [inline backend/src/router.js:50]: Nop

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Wouldn't it be .put and .delete instead of .get?

File: backend/src/router.js (hunk 2 above)
Comment (ThomasDCW) [reply backend/src/router.js]: Yes i do that

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by ilalande — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**